# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/dbus-cpp/hotfix-cmake.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- dbus-cpp-5.0.6/CMakeLists.txt.vanilla	2026-08-08 20:35:14.098802972 +0300
+++ dbus-cpp-5.0.6/CMakeLists.txt	2026-08-08 20:35:43.003467316 +0300
@@ -33,8 +33,8 @@
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
-include(cmake/EnableCoverageReport.cmake)
-include(cmake/PrePush.cmake)
+#include(cmake/EnableCoverageReport.cmake)
+#include(cmake/PrePush.cmake)
 
 include(GNUInstallDirs)
 
@@ -80,9 +80,9 @@
 
 add_subdirectory(data)
 add_subdirectory(doc)
-add_subdirectory(examples)
+#add_subdirectory(examples)
 add_subdirectory(include)
 add_subdirectory(src)
-add_subdirectory(tests)
+#add_subdirectory(tests)
 
-enable_coverage_report(benchmark)
+#enable_coverage_report(benchmark)

